libs/StatusQ/qml/Status/Controls/Navigation/ApplicationSection.qml

```diff
@@ -0,0 +1,12 @@
+import QtQml
+
+/*!
+  An application section with button and content view
+  */
+QtObject {
```

Comment:
Same comment than before, I like the concept of this object. It gives uniformity and will be clarifying when used.